homeassistant/components/sensor/vasttrafik.py

```diff
@@ -46,6 +47,7 @@
         vol.Required(CONF_FROM): cv.string,
         vol.Optional(CONF_DELAY, default=DEFAULT_DELAY): cv.positive_int,
         vol.Optional(CONF_HEADING): cv.string,
+        vol.Optional(CONF_LINES, default=[]): cv.ensure_list_csv,
```

Comment:
We should not take in `csv`. Just make it ensure list.

Reply:
You will also need a validator of what type you expect it to be in the list.

Reply:
OK, will make these changes.
As the lines here usually are integer numbers, but not always, the items will have to be strings.
I have updated the example above as well.